Fst2List.cpp

```diff
@@ -75,6 +79,7 @@ const char* usage_Fst2List =
         "-d, --disable_loop_check: faster execution at the cost of information about loops\r\n"
         "-v, --verbose: prints information during the process\r\n"
         "-r (s|l|x) <L[,R]>: enclose loops in L and R strings as in (c0|...|cn) by Lc0|..|cnR : default null\r\n"
+        "-b <file>, generate dictionary file according the given configuration file\r\n"
```

Comment:
Remove 'according the given configuration file', it's not implemented in this version

Reply:
done